docs/dotnet-five.md

```diff
@@ -0,0 +1,148 @@
+ms.date: 08/31/2020
+ms.topic: overview
+ms.author: dapine
+author: IEvangelist
+---
+
+# The evolution of .NET Core - .NET 5
+
+The .NET ecosystem is being simplified with the evolution of .NET Core. Naming things is hard, but unifying versions of two disparate .NET implementations can be harder. This article details .NET 5, which can be thought of as .NET Core vNext. With .NET Core concluding at version 3.1 and .NET Framework at version 4.8, to avoid ambiguity the next logical version is .NET version 5.0.
+
+The advent of .NET Core has evolved the .NET ecosystem in compelling ways. It matured as an open-source project on GitHub, celebrating community contributions, and humbly improving over time. .NET 5 is cross-platform, allows for side-by-side installation, and continues support of small project files (SDK-style).
+
+### What .NET 5 is not
+
+To be clear, .NET 5 is not a replacement for .NET Framework. There are no plans to port the following workloads from .NET Framework to .NET 5, as such there are recommended alternatives:
+
+| Technology                             | Recommendation                                              |
+|----------------------------------------|-------------------------------------------------------------|
+| Web Forms                              | [ASP.NET Core Blazor](/aspnet/core/blazor)                  |
+| Windows Communication Foundation (WCF) | [gRPC](/aspnet/core/grpc)                                   |
+| Windows Workflow (WF)                  | [Open-source CoreWF](https://github.com/UiPath-Open/corewf) |
+
+## .NET release schedule
+
+A new major version of .NET is planned to release every November. .NET 5 will not have long-term support (LTS), however; .NET 6 in 2021 will be LTS. As major .NET versions ship, even-numbered releases will receive LTS. Minor releases will occur as needed.
+
+:::image type="content" source="media/dotnet-schedule.svg" alt-text=".NET release schedule":::
+
+## What happens to .NET Standard
+
+With .NET 5, .NET Standard is being de-emphasized. New application development can specify the `net5.0` target framework moniker (TFM) for all project types, including class libraries. Sharing code between .NET 5 workloads is simplified in that all you need is the `net5.0` TFM.
+
+However, if you're planning on sharing code between .NET Framework, .NET Core, and .NET 5 workloads - you can do so by specifying `netstandard2.0` as your TFM. For more information, see [How to specify target frameworks](standard/frameworks.md#how-to-specify-target-frameworks).
+
+## Language updates
+
+With .NET 5, the .NET programming languages are continuing to improve.
+
+### C# updates
+
+C# will move forward in lock-step with .NET releases. Developers writing .NET 5 apps will have access to the latest C# version and features. .NET 5 is paired with C# 9. C# 9 brings many new features to the language, here are a few highlights:
```

Comment:
To avoid making promises about the future --

```suggestion
Developers writing .NET 5 apps will have access to the latest C# version and features. .NET 5 is paired with C# 9. C# 9 brings many new features to the language, here are a few highlights:
```